provision/opentofu/modules/rosa/README.md

```diff
@@ -0,0 +1,6 @@
+# ROSA OpenTofu Modules
+In order to use the modules in this directory, it's necessary for the var `token` to be set. This is an OpenShift Cluster
+Manager token, which can be accessed in the [Red Hat Hybrid Cloud Console](https://console.redhat.com/openshift/token).
+
+Instead of providing the token as a variable everytime `tofu` is called, it's possible to set the value via the
+`TF_VAR_token` environment variable.
```

Comment:
`TF_VAR_token` is such a non-descriptive name :-( I don't know if this can be changed. Again, a nitpick.

Reply:
Good point. I've changed the variable to `rhcs_token` so it becomes `TF_VAR_rhcs_token`.